e2e-tests/tests/namespaces.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from "@playwright/test";
+import { HeadlampPage } from "./headlampPage";
+import { NamespacesPage } from "./namespacesPage";
+
+test("create a namespace with the minimal editor then delete it", async ({
+  page,
+}) => {
+  const name = "testing-e2e";
+  const headlampPage = new HeadlampPage(page);
+  await headlampPage.authenticate();
+
+  await page.click('span:has-text("Cluster")');
+  await page.waitForLoadState("load");
```

Comment:
I think this can be removed. When we authenticate, we reload the page and wait for navigation.
```suggestion
```

Reply:
Ok, I've removed them. Thanks.